faucet/valve.py

```diff
@@ -1729,6 +1745,7 @@ class NoviFlowValve(Valve):
     'Netronome': OVSValve,
     'NoviFlow': NoviFlowValve,
     'Open vSwitch': OVSValve,
+    'TFM Open vSwitch': OVSTfmValve,
```

Comment:
I'm just wondering if "Open vSwitch TFM" might make more logical sense. It would follow our "Generic" and "GenericTFM" we already have.

Reply:
Done.